<ui version="4.0" >
 <class>gdalPreferencesWidget</class>
 <widget class="QWidget" name="gdalPreferencesWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="optionsGroupBox" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>44</y>
     <width>573</width>
     <height>195</height>
    </rect>
   </property>
   <property name="title" >
    <string>Configuration options</string>
   </property>
   <widget class="QCheckBox" name="gdalDataCheckBox" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>62</y>
      <width>143</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>GDAL_DATA</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cacheCheckBox" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>29</y>
      <width>143</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>GDAL_CACHEMAX</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>93</y>
      <width>143</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>GDAL_SKIP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>93</y>
      <width>402</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>125</y>
      <width>143</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>GDAL_DRIVER_PATH</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>159</y>
      <width>143</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>OGR_DRIVER_PATH</string>
    </property>
   </widget>
   <widget class="QWidget" name="" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>28</y>
      <width>402</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QSpinBox" name="cacheSpinBox" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="maximum" >
        <number>1000000</number>
       </property>
       <property name="singleStep" >
        <number>10000</number>
       </property>
       <property name="value" >
        <number>40000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="extraOptGroupBox" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>245</y>
     <width>573</width>
     <height>327</height>
    </rect>
   </property>
   <property name="title" >
    <string>Extra optons</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="extraOptVerticalLayout" >
    <item>
     <widget class="QTableWidget" name="extraOptTableWidget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="horizontalScrollMode" >
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <column>
       <property name="text" >
        <string>Key</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Value</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="" >
   <layout class="QHBoxLayout" name="infoHorizontalLayout" >
    <item>
     <spacer name="infoHorizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>188</width>
        <height>24</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="infoButton" >
      <property name="text" >
       <string>Show detailed GDAL info</string>
      </property>
      <property name="icon" >
       <iconset resource="../../resources.qrc" >
        <normaloff>:/images/info.svg</normaloff>:/images/info.svg</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionAddEnvVar" >
   <property name="icon" >
    <iconset resource="../../resources.qrc" >
     <normaloff>:/images/add.svg</normaloff>:/images/add.svg</iconset>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
   <property name="toolTip" >
    <string>Add a new item</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemoveEnvVar" >
   <property name="icon" >
    <iconset resource="../../resources.qrc" >
     <normaloff>:/images/remove.svg</normaloff>:/images/remove.svg</iconset>
   </property>
   <property name="text" >
    <string>Remove</string>
   </property>
   <property name="toolTip" >
    <string>Remove selected items</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEditEnvVar" >
   <property name="icon" >
    <iconset resource="../../resources.qrc" >
     <normaloff>:/images/edit.svg</normaloff>:/images/edit.svg</iconset>
   </property>
   <property name="text" >
    <string>Edit</string>
   </property>
   <property name="toolTip" >
    <string>Edit current item</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionClearEnv" >
   <property name="icon" >
    <iconset resource="../../resources.qrc" >
     <normaloff>:/images/clear.svg</normaloff>:/images/clear.svg</iconset>
   </property>
   <property name="text" >
    <string>Clear</string>
   </property>
   <property name="toolTip" >
    <string>Clear all items</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cacheCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>cacheSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>94</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel" >
     <x>217</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
